MonoGame.Framework/WindowsUniversal/UAPGameWindow.cs

```diff
@@ -241,8 +241,8 @@ private void DisplayProperties_OrientationChanged(DisplayInformation dinfo, obje
 
         protected override void SetTitle(string title)
         {
-            // NOTE: There is no concept of a window 
-            // title in a Metro application.
+            var appView = ApplicationView.GetForCurrentView();
+            appView.Title = title;
```

Comment:
How about we change this to something like:

```
Debug.WriteLine("WARNING: GameWindow.Title has no affect under UWP.")
```

... or something like this.  This way at least the user sees this in the output window when debugging and can be aware of it.